src/main/java/org/folio/ed/service/RemoteStorageService.java

```diff
@@ -29,11 +23,11 @@
   private static final String STAGING_DIRECTOR_NAME = "Dematic_SD";
 
   private final RemoteStorageClient remoteStorageClient;
-  private final SecurityManagerService securityManagerService;
-  private final SystemParametersHolder systemParametersHolder;
+  private final AccessionQueueRecordToAsrItemConverter accessionQueueRecordToAsrItemConverter;
+  private final RetrievalQueueRecordToAsrRequestConverter retrievalQueueRecordToAsrRequestConverter;
 
   public List<AccessionQueueRecord> getAccessionQueueRecords(String storageId) {
-    return remoteStorageClient.getAccessionsByQuery(buildQueryByStorageId(storageId))
+    return remoteStorageClient.getAccessionsByQuery("storageId=" + storageId + "&accessioned=false")
```

Comment:
@siarhei-charniak , please, confirm. This affects your logic for StagingDirector.

Reply:
It's okay, confirmed.